### DISCUSSION TITLE
Start up error

### DISCUSSION THREAD

**Discussion by airstring** [jertel/elastalert2#1056] [Q&A] [answered]
There was an error when starting：
elasticalert-elastalert-1 | elastalert.util.EAException: Error loading file /opt/elastalert/rules/smtp_auth_file.yml: Invalid Rule file: /opt/elastalert/rules/smtp_auth_file.yml
elasticalert-elastalert-1 | 'type' is a required property

cat rules/smtp_auth_file.yml
user: "[zsdf@gmail.com](mailto:zsdf@gmail.com)"
password: "123456"

**Comment by nsano-rururu** [comment, marked as answer]
Please read the project guidelines below and add the information you need for your research. No one can help you in your current state.
https://github.com/jertel/elastalert2/discussions/11
